app/assets/javascripts/components/mentors/_index.js.jsx

```diff
@@ -0,0 +1,11 @@
+var Index = React.createClass({
+  render() {
```

Comment:
Yep! @neight-allen  It's now within a subfolder for 'mentors' but i'll change it to be MentorIndex